com.unity.ml-agents/Documentation~/index.md

```diff
@@ -1 +1,71 @@
-Please see the [ML-Agents README)(https://github.com/Unity-Technologies/ml-agents/blob/master/README.md)
\ No newline at end of file
+# About Unity MLAgents package
+
+The MLAgents package contains the C# SDK for the
+[Unity ML-Agents Toolkit](https://github.com/Unity-Technologies/ml-agents).
+
+The package provides the ability for any Unity scene to be converted into a learning
+environment where character behaviors can be trained using a variety of machine learning
+algorithms. Additionally, it enables any trained behavior to embedded back into the Unity
+scene. More specifically, the package provides the following core functionalities:
+* Define Agents: entities whose behavior will be learned. Agents are entities
+that generate observations (through sensors), take actions and receive rewards from
+the environment.
+* Define Policies: entities that specifiy a behavior. Multiple agents can share the
+same Policy and a scene may have multiple Brains.
+* Record demonstrations of an agent within the Editor. These demonstrations can be
+valuable to train a behavior for that agent.
+* Embedding a trained behavior into the scene. This an Agent can switch from a
+learning brain to an inference brain.
+
+Note that this package does not contain the machine learning algorithms for training
+behaviors. It relies on a Python package to orchestrate the training. This package
+only enables instrumenting a Unity scene and setting it up for training.
```

Comment:
Do we want to reference inference via the Barracuda package?

Reply:
Good call. @surfnerd - is there a way to link to other packages here? We can just reference them by name too.

Reply:
We can link to their docs on the unity site.

Reply:
https://docs.unity3d.com/Packages/com.unity.barracuda@latest/index.html